cms/djangoapps/contentstore/courseware_index.py

```diff
@@ -175,8 +181,31 @@ def index_item(item, skip_index=False, groups_usage_info=None):
                 return
 
             item_content_groups = None
+
+            if item.category == "split_test":
+                user_partitions = item.user_partitions
+                split_partitions = []
+                if user_partitions:
```

Comment:
When we try something like that we get the following error message in console:
`AttributeError: 'SplitTestDescriptorWithMixins' object has no attribute 'description'`